apps/condo/domains/news/schema/NewsItem.js

```diff
@@ -153,6 +153,40 @@ const NewsItem = new GQLListSchema('NewsItem', {
             },
         },
 
+        compactScopes: {
+            schemaDoc: 'Returns the number of scopes that are specified for sending the news, and also the first two of them.\n' +
+                'Used to reduce requests for get of scopes in the UI',
+            type: 'Virtual',
+            extendGraphQLTypes: ['type ShortScopesField { count: Int!, firstTwo: [NewsItemScope]! }'],
+            graphQLReturnType: 'ShortScopesField',
+            resolver: async (item) => {
+                const firstTreeScopes = await itemsQuery('NewsItemScope', {
+                    where: {
+                        newsItem: { id: item.id }, deletedAt: null,
+                    },
+                    first: 3,
```

Comment:
Maybe move the number of scopes in some constant.
```suggestion
                    first: SOME_CONSTANT + 1,
```